images/public-log-vendor-matcher/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM golang:1.16.5-stretch AS godeps
+RUN go get -u github.com/mikefarah/yq/v4 && \
+  test -f /go/bin/yq
```

Comment:
Is there any reason this can't use the same version of yq used by the k8s-infra image?

Reply:
I like single binaries, but that would be easy enough to swap it out.

Reply:
This has been swapped out in cf5172cc47e96bd721a4e983e0a194b28759f83f